src/main/java/com/example/Ingress_lab/assignment/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.Ingress_lab.assignment.controller;
+
+import com.example.Ingress_lab.assignment.entity.UserEntity;
+import com.example.Ingress_lab.assignment.service.UserService;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/v2/users")
+@FieldDefaults( makeFinal = true, level = AccessLevel.PRIVATE)
+public class UserController {
+    UserService userService;
+
+
+    @GetMapping
+    public List<UserEntity> getUserWithAge(@RequestParam(required = false) Integer age){
```

Comment:
[Optional]: 
1. create UserCriteria class, ex: class UserCriteria { private Integer age; }
2. getUserWithAge(UserCriteria userCriteria)
3. use userCriteria.getAge() in service